causal_testing/specification/capabilities.py

```diff
@@ -0,0 +1,75 @@
+"""
+This module contains the Capability and TreatmentSequence classes to implement
+treatment sequences that operate over time.
+"""
+from causal_testing.specification.variable import Variable
+
+
+class Capability:
+    """
+    Data class to encapsulate temporal interventions.
+    """
+
+    def __init__(self, variable: Variable, value: any, start_time: int, end_time: float):
```

Comment:
We should use the [Any](https://docs.python.org/3/library/typing.html#the-any-type) from the Typing library rather than the generic `any`. The reason for this is in theory the generic `any` isnt related to the concept of 'any' object and would fail if we had a static type checker

Reply:
Fixed